caputilities/redfishClient.go

```diff
@@ -106,8 +106,9 @@ func (client *RedfishClient) GetRootService(device *RedfishDevice) error {
 		if err != nil {
 			log.Error(err.Error())
 		}
-		log.Error("Could not retrieve ServiceRoot for " + device.Host + ", got: " + string(body))
-		return nil
+		errMsg := "Could not retrieve ServiceRoot for " + device.Host + ", got: " + string(body)
+		log.Error(errMsg)
+		return fmt.Errorf(errMsg)
```

Comment:
Discussion pending for [Bharath's comment](https://github.com/a-ajith/PluginCiscoACI/pull/12/files#r565213463).